src/components/GroupMenu/GroupMenu.tsx

```diff
@@ -16,10 +16,14 @@ import logoAccount from "@assets/images/logoAccount.png";
 
 import "./GroupMenu.css";
 
+import { useOutsideClick } from "@/hooks/useOutsideClick.tsx";
```

Comment:
```suggestion
import { useOutsideClick } from "@hooks/useOutsideClick.tsx";
```
**Suggestion** : Remet dans l'ordre des imports